sdk/spring/README.md

```diff
@@ -136,7 +136,7 @@ If you’re a Maven user, add our BOM to your pom.xml `<dependencyManagement>` s
         <dependency>
             <groupId>com.azure.spring</groupId>
             <artifactId>spring-cloud-azure-dependencies</artifactId>
-            <version>4.0.0</version>
+            <version>4.2.0</version>
```

Comment:
How about update it automatically by something like `[//]: # ({x-version-update-start;com.azure.spring:spring-cloud-azure-dependencies;dependecy})`?